packages/blocks/src/api/test/validation.js

```diff
@@ -473,6 +473,24 @@ describe( 'validation', () => {
 			expect( console ).toHaveWarned();
 			expect( isEquivalent ).toBe( false );
 		} );
+
+		it( 'should return true when comparing self-closing and normal tags', () => {
```

Comment:
I might like to see some tests of `isClosedByNextToken`, specifically a case(s) covering its `false` return.